app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/OffboardRouterActivityKt.kt

```diff
@@ -110,14 +109,9 @@ class OffboardRouterActivityKt : AppCompatActivity(),
     }
 
     private fun findRoute() {
-        val offboardRouter = MapboxOffboardRouter(
-            NavigationRoute.builder(this)
-        )
-        directionsSession = MapboxDirectionsSession(
-            offboardRouter,
-            this
-        )
-        ifNonNull(directionsSession, origin, destination) { session, originPoint, destinationPoint ->
+        ifNonNull(origin, destination) { originPoint, destinationPoint ->
+            offboardRouter = MapboxOffboardRouter(this)
```

Comment:
are you sure that we'd like to recreate `MapboxOffboardRouter` for each `getRoute` call? looks like it hides issues

Reply:
I'm just mirroring the old implementation, let's tackle the issue in a separate PR.

Reply:
The issue in question https://github.com/mapbox/mapbox-navigation-android/issues/2329.

Reply:
yep, I'm trying to handle this issue :) My point is We shouldn't hide this problem by recreating instance of `MapboxOffboardRouter` each time when new route is required